### PR TITLE
Added EtherWorld to the Community Built Resources

### PR DESCRIPTION
**Pull request by yashkamalchaturvedi**
Found a good-quality educational resource on Ethereum

### CODE REVIEW

File: src/content/learn/index.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ In addition to the information on this page, there are many community-built reso
 - [What’s new in Eth2](https://eth2.news) _A regular newsletter about Ethereum development_
 - [ethresear.ch forum](https://ethresear.ch/) _Deeper technical discussions on Ethereum for scaling and beyond_
 - [ETHGlobal](https://ethglobal.co) _An Ethereum hackathon series - attend one near you!_
+- [EtherWorld](https://etherworld.co/) _An Ethereum educational resource in the form of blogs and video guides_
```
Comment (EtherWorldCo) [inline src/content/learn/index.md:30]:
```suggestion
- [EtherWorld]( https://etherworld.co/tag/ethereum/) _An Ethereum educational resource in the form of blogs and video guides_
```

**Comment by minimalsm** [comment]
Thanks for the PR @yashkamalchaturvedi ✌️.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳! We're currently working on our 2022 POAPs and they'll be released soon.

**Comment by minimalsm** [comment]
@all-contributors please add @yashkamalchaturvedi for docs

**Comment by minimalsm** [comment]
@all-contributors please add @EtherWorldCo for docs

**Pull request merged**